ui/app/components/ui/sender-to-recipient/sender-to-recipient.component.js

```diff
@@ -22,6 +22,7 @@ function SenderAddress ({
   senderName,
   onSenderClick,
   senderAddress,
+  warnUserOnAccountMismatch = true,
```

Comment:
Nit: Conventionally we use `defautProps` to specify defaults. Better to have once place to set defaults than two, and `defaultProps` lets us assign them the same way between functional and class components.

Reply:
The same applies to the `SenderToRecipient` component, though I'm not sure we need the default in two places 🤔 🤷‍♂️

Reply:
I'll remove the defaultProps on the child component, and move them to use the actual defaultProps mechanism. I thought I remembered that default variables like what I did here were what TypeScript liked... memory is fickle.

Reply:
That does sound plausible. Certainly open to changing the convention if it makes the TypeScript migration easier.